src/search.jl

```diff
@@ -1,106 +1,165 @@
 export MPS_from_gates, unique_neighbors
 export MPSControl
+export spectrum
+
+export _apply_bias!
```

Comment:
We should not export "hidden" methods. I we want the user to have access to these, we should rename them

Reply:
I agreed, I have been using them for testing purposes.

Reply:
Please remove the exports and cal them as `SpinGlassPEPS._method`

Reply:
Done